client/extensions/wp-super-cache/directly-cached-files.jsx

```diff
@@ -40,40 +40,40 @@ const DirectlyCachedFiles = ( {
 				</Button>
 			</SectionHeader>
 			<Card className="wp-super-cache__directly-cached-files">
-				{ !! wp_cache_readonly &&
+				{ cache_readonly &&
 				<p>
 				{ translate(
-					'{{strong}}Warning!{{/strong}} You must make %(wp_cache_path)s wp_cache_writable to enable this feature. ' +
+					'{{strong}}Warning!{{/strong}} You must make %(cache_path)s writable to enable this feature. ' +
 					'As this is a security risk, please make it read-only after your page is generated.',
 					{
-						args: { wp_cache_path: wp_cache_path },
+						args: { cache_path: cache_path },
 						components: { strong: <strong /> },
 					}
 				) }
 				</p>
 				}
-				{ !! wp_cache_writable &&
+				{ cache_writable &&
 				<p>
 				{ translate(
-					'{{strong}}Warning!{{/strong}} %(wp_cache_path)s is wp_cache_writable. Please make it wp_cache_readonly after your ' +
+					'{{strong}}Warning!{{/strong}} %(cache_path)s is writable. Please make it readonly after your ' +
 					'page is generated as this is a security risk.',
 					{
-						args: { wp_cache_path: wp_cache_path },
+						args: { cache_path: cache_path },
 						components: { strong: <strong /> },
 					}
 				) }
 				</p>
 				}
 				<p>
 					{ translate(
-						'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This ' +
+						'Directly cached files are files created directly off %(cache_path)s where your blog lives. This ' +
```

Comment:
a8ci18n shouldn't suggest the previous version of the string when it is being changed. Probably confused by the multi-line concatenation here